install-dev/data/xml/tab.xml

```diff
@@ -573,7 +573,7 @@
                     </tab>
                     <tab id="Profiles" id_parent="Employees" active="1" enabled="1">
```

Comment:
```suggestion
                    <tab id="Roles" id_parent="Employees" active="1" enabled="1">
```

Reply:
Is this necessary to change ID? PHP class is still called `Profiles`.

Reply:
You're right! Since the ID is a technical information we'd better keep it in sync with the code and its technical naming as well. Since the controller/entity is Profile let's keep Profiles here